core/server/proxy/src/main/java/alluxio/web/ProxyWebServer.java

```diff
@@ -53,7 +53,8 @@ public ProxyWebServer(String serviceName, InetSocketAddress address,
     super(serviceName, address);
 
     // REST configuration
-    ResourceConfig config = new ResourceConfig().packages("alluxio.proxy", "alluxio.proxy.s3")
+    ResourceConfig config = new ResourceConfig().packages("alluxio.proxy", "alluxio.proxy.s3",
+            "alluxio.proxy.s3.logging")
```

Comment:
Ah... that's why we never saw those logs from `RequestLoggingFilter` before...

Reply:
right, it will not work without this line...